e2e-tests/src/test/rewards.rs

```diff
@@ -40,6 +41,127 @@ fn get_non_reserved_members_for_session(config: &Config, session: SessionIndex)
     non_reserved.iter().map(account_from_keypair).collect()
 }
 
+fn validators_bond_extra_stakes(config: &Config, additional_stakes: Vec<u128>) {
+    let node = &config.node;
+    let root_connection = config.create_root_connection();
+
+    let validator_seeds = get_validators_seeds(config);
+    let controller_seeds: Vec<_> = validator_seeds
+        .iter()
+        .map(|v| format!("{}//Controller", v))
+        .collect();
+
+    let controller_accounts_key_pairs = accounts_seeds_to_keys(&controller_seeds);
+    let stash_accounts_key_pairs = accounts_seeds_to_keys(&validator_seeds);
+
+    let controller_accounts: Vec<AccountId> = controller_accounts_key_pairs
+        .iter()
+        .map(account_from_keypair)
+        .collect();
+
+    balances_batch_transfer(&root_connection.as_signed(), controller_accounts, TOKEN);
+
+    stash_accounts_key_pairs
+        .iter()
+        .zip(additional_stakes.iter())
+        .for_each(|(key, additional_stake)| {
+            let id = account_from_keypair(key);
+            balances_transfer(
+                &root_connection.as_signed(),
+                &id,
+                *additional_stake + TOKEN,
+                XtStatus::InBlock,
+            );
+            let stash_connection = SignedConnection::new(node, key.clone());
+            let xt = stash_connection
+                .as_connection()
+                .staking_bond_extra(*additional_stake);
+            send_xt(&stash_connection, xt, Some("bond_extra"), XtStatus::InBlock);
+        });
+}
+
+pub fn points_stake_change(config: &Config) -> anyhow::Result<()> {
+    const MAX_DIFFERENCE: f64 = 0.05;
+
+    let node = &config.node;
+    let accounts = get_validators_keys(config);
+    let sender = accounts.first().expect("Using default accounts").to_owned();
+    let connection = SignedConnection::new(node, sender);
+    let root_connection = config.create_root_connection();
+
+    let reserved_members: Vec<_> = get_reserved_members(config)
+        .iter()
+        .map(account_from_keypair)
+        .collect();
+    let non_reserved_members: Vec<_> = get_non_reserved_members(config)
+        .iter()
+        .map(account_from_keypair)
+        .collect();
```

Comment:
Suggestion: what about changing these (and twin lines in the other test) to some function like `members_accounts` which would return two vecs?

Reply:
Probably a good idea